src/renderer/api-decorator.js

```diff
@@ -172,13 +172,8 @@ limitations under the License.
     function wireUpZoomEvents() {
         // listen for zoom-in/out keyboard shortcut
         // messages sent from the browser process
-        ipc.on(`zoom-${renderFrameId}`, (event, zoomIn, reset) => {
-            if (reset) {
-                webFrame.setZoomLevel(0);
-            } else {
-                let level = Math.floor(webFrame.getZoomLevel());
-                webFrame.setZoomLevel(zoomIn ? ++level : --level);
-            }
+        ipc.on(`zoom-${renderFrameId}`, (event, zoomIncrement) => {
+            webFrame.setZoomLevel(zoomIncrement ? webFrame.getZoomLevel() + zoomIncrement : 0);
```

Comment:
Is there a reason for getting rid of `Math.floor()`? Usually when you get the zoom level it doesn't come back as an integer value.

Reply:
yeah [it does](https://github.com/electron/electron/blob/master/docs/api/web-contents.md#contentsgetzoomlevelcallback) (come back as a number)

Reply:
@HarsimranSingh yeah [it does](https://github.com/electron/electron/blob/master/docs/api/web-contents.md#contentsgetzoomlevelcallback)! (come back as a number)

Reply:
Will add it back in.